include/apps/progs.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * WARNING: do not edit!
```

Comment:
Did you add this generated file deliberately? Since it's purely declarations, it probably doesn't need guards, but I'd still add them.

Reply:
No, I didn't mean to add that.  I did `rebase -i` and removed the file from the commit. It was a mistake, confused with #173.  I added a fixup commit to `apps/progs.pl` that *does* add the guards, if only for consistency.